unpacktool/unpacktool.cpp

```diff
@@ -1,4 +1,17 @@
 #include <stdio.h>
+#include <string.h>
+
+#include <string>
+#include <vector>
+
+#ifdef _WIN32
+#include <Windows.h>
+#include <shellapi.h>
```

Comment:
You can just delete the Windows.h and shellapi.h includes, they're not actually used any more

Reply:
Done.